lib/__tests__/disableRanges.test.js

```diff
@@ -238,6 +238,24 @@ it('disable single line multiple rules', () => {
 	);
 });
 
+it('disable single line one rule and re-enable all', () => {
+	return testDisableRanges(
+		'a {} /* stylelint-disable-line block-no-empty */\n/* stylelint-enable */',
+	).then((result) => {
+		expect(result.stylelint.disabledRanges).toEqual({
```

Comment:
We must expect error here since there is nothing to enable, if I understand correct condition with `No rules have been disabled` error =)

Reply:
You're right :)

Reply:
Hah, I found a bug in ranges assignment, I will do fix